_projects/2023/100407614/100407614.Rmd

```diff
@@ -0,0 +1,581 @@
+---
+title: "Analyzing the Relationship between Cigarettes Sales and Lung Cancer Mortality in the US"
+description: |
+  Every year, around 8 million people die prematurely as a result of smoking. In this case, examining the relationship across the cigarettes sales in men in the United States.
+categories: "2023"
+author: Andrea Baños Ramos
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
```

Comment:
Please add a short introduction presenting some context: the graph that is going to be replicated, the source, and why you chose it.